packages/validator/src/ui-validator.ts

```diff
@@ -188,7 +188,7 @@ export class UiValidator {
   }
 
   private isEquals(expected: string | number, value: unknown): boolean {
-    return value === expected;
+    return value == expected;
```

Comment:
Softening this check as it was failing when we were checking "1" === 1, as the text inputs returns a string, we are going with a relax check.